### DISCUSSION TITLE
Does MAUI have controls similar to Spacer of SwiftUI?

### DISCUSSION THREAD

**Discussion by SF-Simon** [dotnet/maui#14049] [Q&A]
![image](https://user-images.githubusercontent.com/11264289/226148128-ba76e477-f060-407f-8ffc-03f2ebe2826f.png)

[https://developer.apple.com/documentation/swiftui/spacer](https://developer.apple.com/documentation/swiftui/spacer)
A simple understanding is to maximize the height or width of the parent element within any layout component, while limiting the minimum height and width.

**Comment by xtuzy** [comment]
no, you can use layoutoptions = left center right to set position in maui.

**Reply by SF-Simon** [reply]
The two effects are still different. Spacer can flexibly control the width of the parent node, while LayoutOptions can only control the position of the child node.

**Reply by xtuzy** [reply]
if you want maximize layout, if you want max h and w, you can use grid, if you want only max w, use verticalstacklayout, if you want max h, use horizentalstacklayout.

**Comment by saint4eva** [comment]
NO, there is none in Maui.Core and Maui Controls. But if you are using Comet C# MVU, there is a control called Spacer - it adds space above and below a Views

**Reply by LuohuaRain** [reply]
I am curious about when Comet will be officially supported?

**Reply by saint4eva** [reply]
@davidortinau David can answer that.
